polymarket-subgraph/src/FixedProductMarketMakerMapping.ts

```diff
@@ -274,7 +274,7 @@ export function handleBuy(event: FPMMBuy): void {
     collateralScaleDec,
     TRADE_TYPE_BUY,
   );
-  updateMarketPositionFromTrade(event);
+  // updateMarketPositionFromTrade(event);
```

Comment:
This will mean historical pnl from the fpmm era will not exist anymore. Is that expected?